lib/fizzy/types.hpp

```diff
@@ -363,10 +363,6 @@ struct Code
     // The instructions bytecode without immediate values.
     // https://webassembly.github.io/spec/core/binary/instructions.html
     std::vector<uint8_t> instructions;
```

Comment:
I also tried using `bytes`, but the performance is the same. So I'm happy to leave vector as it is simpler internally (string has SSO which is useless in this case).